src/bio/bio_xstream.c

```diff
@@ -1395,16 +1473,25 @@ bio_xsctxt_alloc(struct bio_xs_context **pctxt, int tgt_id, bool self_polling)
 	}
 
 	/* Initialize per-xstream blobstore context */
-	rc = init_blobstore_ctxt(ctxt, tgt_id);
-	if (rc)
-		goto out;
+	for (st = SMD_DEV_TYPE_DATA; st < SMD_DEV_TYPE_MAX; st++) {
+		/* only data blob now */
```

Comment:
Supporting other blobs will be implemented in follow-on patch? Then the logic of allocating 'bxb' in init_xs_blobstore_ctxt() needs be revised later.